src/core/api.ts

```diff
@@ -78,31 +93,31 @@ export function createApi(
     return {
         subscribe,
         unsubscribe,
-        // TODO: Support assemblies (we can infer this from the spec)
         zoomTo: (viewId, position, padding = 0, duration = 1000) => {
             // Accepted input: 'chr1' or 'chr1:1-1000'
             if (!position.includes('chr')) {
                 console.warn('Genomic interval you entered is not in a correct form.');
                 return;
             }
 
+            const assembly = getTrack(viewId)?.spec.assembly;
             const chr = position.split(':')[0];
-            const chrStart = GET_CHROM_SIZES().interval?.[chr]?.[0];
+            const chrStart = GET_CHROM_SIZES(assembly).interval?.[chr]?.[0];
```

Comment:
We will need to touch this part when refactoring the assembly

Reply:
If we absolutely need this info, you should extract it once rather than calling `GET_CHROM_SIZES` multiple times in this funciton.

```typescript
            const chrStart = GET_CHROM_SIZES(assembly).interval?.[chr]?.[0];
```